### ISSUE TITLE
build against libzrtpcpp 4 fails

### ISSUE THREAD

**Issue by mkubecek** [LubosD/twinkle#12]
A build against zrtpcpp library version 4 (available and default e.g. in openSUSE 13.2 and openSUSE Factory) fails. The obvious problem is that <libzrtpcpp/zrtpccrtp.h> now (since version 3.0.0) includes <ZrtpQueue.h> rather than <libzrtpcpp/ZrtpQueue.h>. There might be more subtle incompatibilities, however.

This is not really pressing yet as even openSUSE Factory still provides zrtpcpp2-devel package to build against zrtpcpp 2.

**Comment by LubosD** [comment]
Hmm, I just tried it under Gentoo and Twinkle doesn't build with the library build *as configured by Gentoo*. But if I compile my own libzrtpcpp 4.2.4 with ````-DCCRTP=On````, then Twinkle builds and links just fine.

So it's possible SUSE made the mistake of not enabling all the options libzrtpcpp has.

**Comment by mkubecek** [comment]
Ah, markdown ate important parts of the key sentence. Let me try again: The obvious problem is that zrtpccrtp.h now (since version 3.0.0) includes `<ZrtpQueue.h>` rather than `<libzrtpcpp/ZrtpQueue.h>` (as it does in version 2). Therefore we should compile (preprocess) with `-I /usr/include/libzrtpcpp`. Using `pkg-config --cflags libzrtpcpp` should work for both.

**Comment by mkubecek** [comment]
Thanks, now it builds against both versions (2 and 4). Closing.
